pandas/core/indexes/accessors.py

```diff
@@ -102,7 +102,9 @@ def _delegate_property_get(self, name):
         else:
             index = self._parent.index
         # return the result as a Series, which is by definition a copy
-        result = Series(result, index=index, name=self.name).__finalize__(self._parent)
+        result = Series(result, index=index, name=self.name, copy=False).__finalize__(
+            self._parent
+        )
 
         # setting this object will show a SettingWithCopyWarning/Error
```

Comment:
Sidenote: seeing this here, this might also be a case to look into if we can raise a "chained assignment" error